### PR TITLE
💥 Upgrade to what I demoed at DDD Brisbane

### PR DESCRIPTION
**Pull request by Hona**

### CODE REVIEW

File: src/VerticalSliceArchitecture/Features/Todos/Commands/CreateTodo/CreateTodoEndpoint.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public static async Task<IResult> HandleAsync(CreateTodoRequest request, ITodoRepository todoRepository, CancellationToken cancellationToken)
+    {
+        var output = new Todo
+        {
+            Text = request.Text
+        };
+
+        await todoRepository.AddAsync(output, cancellationToken);
+
+        return Results.Created("/todo", output.Id);
```
Comment (danielmackay) [inline src/VerticalSliceArchitecture/Features/Todos/Commands/CreateTodo/CreateTodoEndpoint.cs:17]: Check that this correctly sets the `location` header

**Review by danielmackay — APPROVED** [review]
LGTM 💪

**Review by danielmackay — APPROVED** [review]
LGTM 💪

**Review by danielmackay — APPROVED** [review]

File: src/VerticalSliceArchitecture/Common/BaseEntity.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace VerticalSliceArchitecture.Common;
+
+public abstract class BaseEntity
+{
+    public readonly List<INotification> StagedEvents = new();
```
[the remark this answers is not in the document]
Comment (Hona) [reply src/VerticalSliceArchitecture/Common/BaseEntity.cs:5]: Will check on this in future

File: src/VerticalSliceArchitecture/Features/Todos/Commands/CreateTodo/CreateTodoEndpoint.cs (hunk 1 above)
Comment (Hona) [reply src/VerticalSliceArchitecture/Features/Todos/Commands/CreateTodo/CreateTodoEndpoint.cs:17]: Will iterate on this later - I agree that this is probably not correct

**Pull request merged**